packages/react/src/components/cookieConsentCore/siteSettingsEditor/cookie-banner-admin-ui.js

```diff
@@ -0,0 +1,318 @@
+function cookieBannerAdmin() {
+  // Configuration and state used in the script
+  const config = {
+    textarea: document.getElementById('edit-site-settings'),
+    bannerElement: document.getElementById('editor_holder'),
+    errorHolderElement: document.getElementById('error_holder'),
+    errorHolderElement2: document.getElementById('error_holder2'),
+    errorHolderElement3: document.getElementById('error_holder3'),
+    editorOptions: {
+      theme: 'bootstrap3',
+      iconlib: 'bootstrap',
+      show_opt_in: true,
+      disable_edit_json: true,
+      disable_properties: true,
+      disable_array_delete_all_rows: true,
+      disable_array_delete_last_row: true,
+      keep_oneof_values: false,
+      prompt_before_delete: true,
+    },
+    defaultLanguages: [
+      { code: 'fi', name: 'Finnish', direction: 'ltr' },
+      { code: 'sv', name: 'Swedish', direction: 'ltr' },
+      { code: 'en', name: 'English', direction: 'ltr' },
+    ],
+    saveButton: document.querySelector('#edit-submit'),
+  };
+
+  // state used in the script
+  const state = {
+    jsonValue: {},
+    languages: [],
+    schema: {},
+    bannerEditor: null,
+    textareaErrors: [],
+    editorErrors: [],
+  };
+
+  /**
+   * Utility to debounce a function
+   * @param {function} func Function to be debounced
+   * @param {number} delay Delay in ms for the debounce
+   * @returns {function} The debounced function
+   */
+  function debounce(func, delay) {
+    let timeoutId;
+    return (...args) => {
+      clearTimeout(timeoutId);
+      timeoutId = setTimeout(() => func(...args), delay);
+    };
+  }
+
```

Comment:
`Lodash` is in HDS, which has the same function. Better to use that than create own.

Reply:
https://www.npmjs.com/package/lodash says that the unpacked size is 1.41 MB.

This folder contains a standalone approach to editing the json, purely optional version.

I really do not think that loading 1.41MB to replace these 7 rows of code is smart in this case where I'm trying to show how these features can be included in a random CMS.

If this code would be in the actual package of HDS for example in cookieConsentCore and built, I would definitely agree, using lodash and other libraries in use would be smart.  But as these files work without a build step (on purpose), I do not think that it's a smart move here.

Reply:
Oh yes, if this not part of the install, then no need to add libraries.